client/main.js

```diff
@@ -37,3 +38,13 @@ AutoForm.addHooks(['partyForm'],{
     FlowRouter.go('/party/' + this.docId);
   }
 });
+
+Meteor.subscribe('profiles');
+
+Meteor.setInterval(() => {
+  if (Meteor.loggingIn() || Meteor.user()) {
+    if (Profiles.findOne({userId: Meteor.userId()}).banned) {
+      Meteor.logout();
```

Comment:
Could you add a Bert alert here to say something like "Login denied. This account is currently disabled", please?